python-package/lightgbm/libpath.py

```diff
@@ -20,7 +20,8 @@ def find_lib_path():
         dll_path.append(os.path.join(curr_path, '../../windows/x64/Dll/'))
         dll_path.append(os.path.join(curr_path, './windows/x64/Dll/'))
         dll_path.append(os.path.join(curr_path, '../../Release/'))
-        dll_path = [os.path.join(p, 'lib_lightgbm.dll') for p in dll_path]
+        dll_path.append(os.path.join(curr_path, '../../windows/x64/Release/')) 
+        dll_path = [os.path.join(p, 'lightgbm.dll') for p in dll_path]
```

Comment:
No. The default dll name is ```lib_lightgbm.dll``` and in ```x64/Dll/```. 
Why you build it to the ```Release``` folder and change its name?

You will break other users build.